### ISSUE TITLE
MinSDK is 14 instead of 7

### ISSUE THREAD

**Issue by ultrah** [timehop/sticky-headers-recyclerview#25]
If I add this library to a project with minSDK 10, I get this:

`Error:Gradle: Execution failed for task ':app:processAppAlphaManifest'.
    > Manifest merger failed : uses-sdk:minSdkVersion 10 cannot be smaller than version 14 declared in     library /.../app/build/intermediates/exploded-  aar/com.timehop.stickyheadersrecyclerview/library/0.3.6/AndroidManifest.xml

This can easily be fixed by `<uses-sdk tools:overrideLibrary="com.timehop.stickyheadersrecyclerview"/>` but maybe you can change it anyway.

**Comment by jacobtabak** [comment]
Does the library work OK for you on SDK 10?  I just tried to run the sample on SDK 10 and the headers weren't measured correctly.  Not sure if it's worth figuring out why.

**Comment by ultrah** [comment]
TBO I forgot to check if actually looks ok. I did now and the headers fill the whole screen. If you are not planning on it, I will probably have a look and try to fix it.

**Comment by jacobtabak** [comment]
That would be great - I spent a few minutes looking at it this morning but didn't find the issue.  It looks like the headers are [measured and laid out correctly](https://github.com/timehop/sticky-headers-recyclerview/blob/master/library/src/main/java/com/timehop/stickyheadersrecyclerview/StickyRecyclerHeadersDecoration.java#L181-L182) but not drawn correctly.

**Comment by ultrah** [comment]
Yes, it looks like the the header view itself is not the issue. Even just rendering a fixed sized view directly onto the canvas without translation fills the canvas with that view:

    canvas.save();
    mAdapter.onCreateHeaderViewHolder(parent).itemView.draw(canvas);
    canvas.restore();

Not sure what could be done here.

**Comment by Shyish** [comment]
There are some other libraries working for API 7, so if you don´t plan to support such version it would be nice to change the readme: https://github.com/timehop/sticky-headers-recyclerview/pull/40

Personally I would like to try this lib since it seems easier to use than others (e.g. https://github.com/eowise/recyclerview-stickyheaders) but I cannot increase that much the minimum sdk of my app :/

**Comment by jacobtabak** [comment]
I think this should be able to work on 7+ with a few modifications. I tried
earlier but didn't have much time and couldn't find a solution. I will
merge your PR for now but it would be awesome if someone else could spend a
few minutes trying to make it work on lower API versions!

**Comment by ThanosFisherman** [comment]
Hey, Can I use this library on minSdkVersion 9 ? Do I have to do those "hacks" you mentioned above?

**Comment by jacobtabak** [comment]
I couldn't figure out how to get it to render correctly but I'm sure it's
possible. Would love a community contribution.

**Comment by ThanosFisherman** [comment]
Thanks I'll try this out and post back if I find any issues on Gingerbreads

**Comment by asmenter15** [comment]
Any feedback on this? Looking to try this out, but figured I would see if there were any immediate issues supporting minSdkVersion 9 or 10.

**Comment by jacobtabak** [comment]
There are definitely still issues - would love for someone to dig in and figure it out.
